main.go

```diff
@@ -1,16 +1,21 @@
 package main
 
-import "fmt"
+import (
+	"fmt"
+
+	"github.com/hoshina85/test-ci/lib"
+	"github.com/serenize/snaker"
```

Comment:
__[gometalinter]__
* [gotype] could not import github.com/serenize/snaker (can't find import: github.com/crifff/test-ci/vendor/github.com/serenize/snaker)

<!-- Comment by [SideCI](https://sideci.com) -->